src/borg/crypto/key.py

```diff
@@ -442,6 +445,18 @@ def env_passcommand(cls, default=None):
                 raise PasscommandFailure(e)
             return cls(passphrase.rstrip('\n'))
 
+    @classmethod
+    def fd_passphrase(cls):
+        try:
+            fd = int(os.environ.get('BORG_PASSPHRASE_FD', None))
```

Comment:
`None` is the default for the 2nd `.get()` argument, so you do not need to give it.

Reply:
Done.